src/test/java/testsuite/integration/hikariCP/HikariCPSQLException.java

```diff
@@ -0,0 +1,18 @@
+package testsuite.integration.hikariCP;
+
+import com.zaxxer.hikari.SQLExceptionOverride;
+
+import java.sql.SQLException;
+
+public class HikariCPSQLException implements SQLExceptionOverride {
```

Comment:
We need a section about connection pools in general and HikariCP in particular to the driver user docs. A new chapter to Failover Plugin would be fine. I'd add this class as a sample as well.